ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -1740,12 +1739,24 @@ tests:
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
       E2E_RUN_TAGS: '@amd64 and @azure-ipi and @network-ovnkubernetes and not @fips'
-      OS_IMAGE_MASTERS: azureopenshift:aro4:413_v2:413.92.20230614
-      OS_IMAGE_MASTERS_PLAN: NoPurchasePlan
+      OS_IMAGE_VERSION: 413.92.2023101700
       TAG_VERSION: '@4.15'
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-marketplace
+- as: azure-ipi-marketplace-noplan-f28
```

Comment:
How about merge the coverage into the above destructive ci? non-destructive cover `413.92.2023101700`, destructive one covers `azureopenshift:aro4:413_v2:413.92.20230614` ?